Edits for plots for paper (#9)
ebmdatalab/scmd-narcolepsy: README.md

Reviewer: looks good! i'd add relative paths to all the files and more information where the data came from

# scmd-narcolepsy
Investigation of medicines used in Primary (English Prescribing Data) care & secondary (Secondary Care Medicines Data) care to treat narcolepsy

Description of R scripts:

>   - "OctGitHubFinal.R" - Data cleaning + analysis of secondary care data (SCMD). Largely adapated from initial code provided by DataLab team

Reviewer: use relative file paths here and backticks instead of quotation marks to specify the files. also no need to say that the code came from us originally I think. I'd keep it simple and just say something like this:

```suggestion
  - `scripts/OctGitHubFinal.R` - Data cleaning and analysis of secondary care data (SCMD)
```

  - "OctGitMapping.R" - Incoporation of SCMD with geographic data to plot volumes of DDDs per ICB on map of ICBs + using separate data frames
  - "OctGitPrimaryCare.R" - Data Cleaning + creating maps for primary care (English prescribing data) mapped to ICB regions
  - "OctGitHubPrimaryReformatCode.R" - Data Cleaning/Reformatting primary care data to facilitate create equivalent plots to secondary care plots (National 3 monthly rolling average, volumes per geographic region, Top ICBs prescribing)

Description of Datasets:
Raw data (.csv):

>   - "epdall" - Primary care ("English Prescribing Data") data for all prescriptions of Sodium Oxybate, Pitolisant, Solriamfetol issued between Jan 2019 - Dec 2022. Excluding prescriptions which could not be mapped to ICBs due to incorrect codes

Reviewer: could you add the source of the data somewhere, maybe a link where you downloaded the datasets in this list?
